### PR TITLE
Fix deployment on prod sprint 53

### PR DESCRIPTION
**Pull request by stewal19**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one test exists testing the new feature
  - [x] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed
  - [ ] The main branch is deployed to a dev server with `Reset non-user related Docker Volumes & Re-populate` turned on
  - [ ] It's verified that the CD run is green
  - [ ] Immediately after that the feature branch is deployed to the same server with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the dev server
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server
- [x] Make sure that pull requests in the development tools repository, which are connected to this pull requests, 
are merged

### CODE REVIEW

File: dataland-internal-storage/src/test/kotlin/db/migration/V10__MigrateRefactoredDataPointClassesLksgTest.kt [hunk 1]
```diff
@@ -11,5 +11,10 @@ class V10__MigrateRefactoredDataPointClassesLksgTest {
             "V10/expectedTransformedDatabaseEntry.json",
             V10__MigrateRefactoredDataPointClassesLksg()::migrateRefactoredDataPointClasses,
         )
+        TestUtils().testMigrationOfSingleDataset(
+            "V10/oldOriginalDatabaseEntry2.json",
```
Comment (SiegfriedSobkowiak) [inline dataland-internal-storage/src/test/kotlin/db/migration/V10__MigrateRefactoredDataPointClassesLksgTest.kt:15]: Die zusätzlichen Testdaten sollten etwas sprechendere Namen erhalten die widerspiegeln warum sie hinzugefügt wurden.

File: dataland-internal-storage/src/test/kotlin/db/migration/V11__MigrateRefactoredDataPointClassesEuTaxoFinancialsTest.kt [hunk 2]
```diff
@@ -11,5 +11,15 @@ class V11__MigrateRefactoredDataPointClassesEuTaxoFinancialsTest {
             "V11/transformedDatabaseEntryFromBackendResponse.json",
             V11__MigrateRefactoredDataPointClassesEuTaxoFinancials()::migrateRefactoredDataPointClasses,
         )
+        TestUtils().testMigrationOfSingleDataset(
+            "V11/oldDatabaseEntryFromBackendResponse2.json",
```
Comment (SiegfriedSobkowiak) [inline dataland-internal-storage/src/test/kotlin/db/migration/V11__MigrateRefactoredDataPointClassesEuTaxoFinancialsTest.kt:15]: Auch hier die Namen etwas unglücklich.

File: dataland-internal-storage/src/test/kotlin/db/migration/V10__MigrateRefactoredDataPointClassesLksgTest.kt (hunk 1 above)
Comment (marcelrasbachdfine) [reply dataland-internal-storage/src/test/kotlin/db/migration/V10__MigrateRefactoredDataPointClassesLksgTest.kt]: fixed

File: dataland-internal-storage/src/test/kotlin/db/migration/V11__MigrateRefactoredDataPointClassesEuTaxoFinancialsTest.kt (hunk 2 above)
Comment (marcelrasbachdfine) [reply dataland-internal-storage/src/test/kotlin/db/migration/V11__MigrateRefactoredDataPointClassesEuTaxoFinancialsTest.kt]: fixed

Comment (marcelrasbachdfine) [reply dataland-internal-storage/src/test/kotlin/db/migration/V11__MigrateRefactoredDataPointClassesEuTaxoFinancialsTest.kt]: just uploaded one refactored version

**Pull request merged**